1.6/Dockerfile.windows

```diff
@@ -0,0 +1,12 @@
+FROM windowsservercore
```

Comment:
Maybe with a Hyper-V  "Linux container": `docker run -it --rm --isolation=hyperv ubuntu` :smile: